public/ipc.js

```diff
@@ -157,17 +155,39 @@ ipc.on('get job', async (event, data) => {
 })
 
 ipc.on('abort job', (event, data) => {
-  Object.values(queues[data.job.queue.name].childPool.retained).forEach((child) => {
-    child.send({__abortJobId__: data.job.id})
-  })
+  message.showMessageBox(
+    {
+      type: 'question',
+      buttons: ['Cancel', 'Abort'],
+      message: `Are you sure you would like to abort transcoding "${data['job']['data'].videoPath}" to ${data['job']['data'].raster} ?`
```

Comment:
Let's change this to `"Are you sure you want to abort ..."`. Also would that break if the task doesn't have a `videoPath` or `raster` in its data?